src/pipeline.ml

```diff
@@ -204,8 +204,8 @@ let v ?app ?notify:channel ?filter ~sched ~staging_auth () =
       docker "docker/storage/Dockerfile"  ["live", "ocurrent/docs-ci-storage-server:live", [`Ci6 "infra_storage-server"]];
     ];
     ocurrent, "current-bench", [
-      docker "pipeline/Dockerfile" ["live", "ocurrent/current-bench-pipeline:live", [`Cb "current-bench_pipeline"]];
-      docker "frontend/Dockerfile" ["live", "ocurrent/current-bench-frontend:live", [`Cb "current-bench_frontend"]];
+      docker "pipeline/Dockerfile" ["live", "ocurrent/current-bench-pipeline:live", [`Cb "current_bench_pipeline"]];
+      docker "frontend/Dockerfile" ["live", "ocurrent/current-bench-frontend:live", [`Cb "current_bench_frontend"]];
```

Comment:
@gs0510 Do you know what these services are called on `packet_current_bench` ?

Reply:
```suggestion
      docker "frontend/Dockerfile" ["live", "ocurrent/current-bench-frontend:live", [`Cb "packet-current-bench_frontend"]];
```